source/jormungandr/tests/routing_tests_experimental.py

```diff
@@ -309,7 +309,7 @@ class TestDistributedTimeFrameDuration(JourneysTimeFrameDuration, NewDefaultScen
     pass
 
 
-def _make_function(from_coord, to_coord, mode, op):
+def _make_function_upper_limit(from_coord, to_coord, mode, op):
     def test_max_mode_direct_path_duration(self):
```

Comment:
If possible, remove the functor would help read the tests.

Also naming suggestion:
```suggestion
    def test_ko_direct_path_longer_than_max_mode_direct_path_duration(self):
```

Reply:
i renamed this in the newest commit